src/include/libminiasync/vdm.h

```diff
@@ -35,6 +35,12 @@ enum vdm_operation_type {
 	VDM_OPERATION_MEMMOVE,
 };
 
+enum vdm_operation_result {
+	VDM_SUCCESS,
```

Comment:
You could add a test that reads the output.

Reply:
I've added a simple check to an existing test and wrote a new one that verifies error states.